framework/CHANGELOG.md

```diff
@@ -4,9 +4,9 @@ Yii Framework 2 Change Log
 2.0.23 under development
 ------------------------
 
-- Bug #17413, #17418: Fixed MSSQL tests (alexkart)
-- Bug #17413: Fixed MSSQL tests: `QueryBuilderTest::testGetColumnType()`, `QueryBuilderTest::testResetSequence()` (alexkart)
+- Bug #17413, #17418, #17426, #17431: Fixed MSSQL tests (alexkart)
 - Bug #17420: Fixed loading of column default values for MSSQL (alexkart)
+- Bug #17395: Fixed issues with actions that contain underscores in their names (alexkart)
```

Comment:
I don't think this one should be moved.

Reply:
Why? It was merged after 2.0.22 release. Am I wrong?

Reply:
Ah, right!